.changeset/cruel-terms-smell.md

```diff
@@ -0,0 +1,6 @@
+---
+"@cloudflare/unenv-preset": patch
+"wrangler": patch
```

Comment:
wrangler should not be listed (only tests are updated and they are not part of the release)
```suggestion
```